libs/runtime/ebpf_hash_table.h

```diff
@@ -56,7 +59,8 @@ extern "C"
         size_t value_size; //< Size of value in bytes.
         // Optional fields.
         ebpf_hash_table_extract_function extract_function; //< Function to extract key from stored value.
-        ebpf_hash_table_allocate allocate; //< Function to allocate memory - defaults to ebpf_epoch_allocate.
+        ebpf_hash_table_allocate allocate; //< Function to allocate memory - defaults to ebpf_epoch_allocate_with_tag.
+        uint32_t allocate_tag;             //< Pool tag to use for allocations - defaults to EBPF_POOL_TAG_EPOCH.
```

Comment:
```suggestion
        uint32_t allocation_tag;             //< Pool tag to use for allocations - defaults to EBPF_POOL_TAG_EPOCH.
```
Also what does "defaults to" mean?  Does it mean when 0 is passed then it's treated as EBPF_POOL_TAG_EPOCH? Clarify, especially since platform_unit_test.cpp has:
`
        .allocate_tag = EBPF_POOL_TAG_DEFAULT,
`
not
`
        .allocate_tag = 0,
`
hence my confusion.  Is this comment saying that if you pass EBPF_POOL_TAG_DEFAULT it will instead use EBPF_POOL_TAG_EPOCH?

Reply:
`allocate_tag == 0` would result in `EBPF_POOL_TAG_EPOCH`. Added a test case and changed comment.